src/theme/colour-semantic/theme-helper.ts

```diff
@@ -141,4 +129,22 @@ export const ColourSemantic = {
     "hyperlink-inverse": getSemanticColour("hyperlink-inverse"),
     "focus-ring": getSemanticColour("focus-ring"),
     "focus-ring-inverse": getSemanticColour("focus-ring-inverse"),
+
+    // new changes:
```

Comment:
no need to include this comment. again can help to reorder the keys to match Figma